### PR TITLE
Update 2017-03-27-multiple-php-versions-the-easy-way.md

### PR DESCRIPTION
**Pull request by ondraondra81**
update to php 7.1 version

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for update Ondra! I'd like to merge this, but I don't have Windows

Can you check it please @jkuchar @enumag?

File: source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md [hunk 1]
```diff
@@ -75,32 +83,24 @@ Fortunately there are packages directly from a maintainer of Debian's PHP packag
...
 (There is also a PPA repository in case you'd rather use Ubuntu instead of Debian.)
 
-We'll now add his repository (as well as enable HTTPS for APT and register the APT key):
-```
-$ apt-get install apt-transport-https
-$ curl https://packages.sury.org/php/apt.gpg | apt-key add -
-$ echo 'deb https://packages.sury.org/php/ stretch main' > /etc/apt/sources.list.d/deb.sury.org.list
-$ apt-get update
-```
-
```
Comment (TomasVotruba) [inline source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md]: Just curious, why did you move this up?

**Comment by enumag** [comment]
@TomasVotruba Huh? This is clearly meant for Debian only.

**Comment by TomasVotruba** [comment]
@enumag Ah, my bad :rofl: I confused it with another Windows post.

**Comment by TomasVotruba** [comment]
@Majkl578 Let me know, if it's ok for you

**Comment by Majkl578** [comment]
No it is not ok.
7.0 is default and the only version in official Debian repository which is the reason why it's used in the article instead of 7.1, to show how official package could be used with different one.

And next time don't ping me here, it won't have effect as I have this repository set as ignored after previous incidents with @TomasVotruba. But I am on Slack. :) Thanks.

**Comment by TomasVotruba** [comment]
@Majkl578 I've updated your comment to remove hate. You're crossing line too far. 

As for post, I'll respect your decision and keep it for 7.0.

@ondraondra81 Let me know, if you'd like to create a new post related to your issue.

File: source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md (hunk 1 above)
Comment (ondraondra81) [inline source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md]: Because without this next commands not working (does matter when try with 7.0 or 7.1)

**Comment by ondraondra81** [comment]
@Majkl578  on clean debian 8 instalation apt-get install php7.0-cli php7.0-fpm not work until Im add deb.sury.org to source list. 
But as you wish...

File: source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md (hunk 1 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md]: I see. Could you send just PR for this change?

Comment (ondraondra81) [inline source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md]: I can do it but I'm not sure that author of the article want it.

Comment (TomasVotruba) [inline source/_posts/2017/2017-03-27-multiple-php-versions-the-easy-way.md]: All right then. Let me know if you'd like to create a new post, that would meet your needs. I can help you with that.
